nodetypes/radon.nodes.docker/DockerEngine/NodeType.tosca

```diff
@@ -11,6 +11,11 @@ node_types:
       port:
         type: integer
         default: { get_attribute: [ SELF, docker, port ] }
+    requirements:
+      - host:
```

Comment:
@sydkbc with this requirement, the DockerEngine node can only be hosted on EC2 now... Was there any particular reason to have such a restriction?

Reply:
That was a requirements for the deployment on xOpera SaaS. If you scroll below you can see that I use the ssh user and key to deploy docker engine. Hence I need to retrieve these properties from the docker engine host, which in my case is an EC2. 

To sum up the requirement is there to acquire ssh_user and ssh_key.